### PR TITLE
Integrate Yahpo to blackbox repository, add example to show how to ru…

### PR DESCRIPTION
**Pull request by geoalgo**
…n ASHA on three epoch-based benchmarks

*Issue #, if available:* https://github.com/awslabs/syne-tune/issues/326

*Description of changes:* 
* Make Yahpo benchmarks available in syne-tune and integrate them in blackbox-repository for easy access. 
* One example showing how to run ASHA on lcbench, fcnet and Nas301 benchmarks

This work is partly based on https://github.com/awslabs/syne-tune/pull/301, I opened a new PR for simplicity given a few merge conflicts.

@pfistl: I cannot add you as a reviewer because you are not in the project, could you still review the PR?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    def time_attribute(self) -> str:
+        """Name of the time column"""
+        return self.benchmark.config.runtime_name
+
+
+def cs_to_synetune(config_space):
+    """
+    Convert ConfigSpace.ConfigSpace to a synetune configspace.
+
+    This should probably either be improved or we could also dump the 'synetune-configspaces' in 'yahpo_data'
```
Comment (aaronkl) [inline syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py:151]: Can we mark this as a FIXME / TODO in the code?

File: syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    like we did for R (paradox) configspaces.
+    Currently this does not cover all possible hyperparameters I think.
+    """
+    hps = config_space.get_hyperparameters()
+
+    keys = []
+    vals = []
+    for a in hps:
+        keys += [a.name]
+        if isinstance(a, ConfigSpace.hyperparameters.CategoricalHyperparameter):
```
Comment (aaronkl) [inline syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py:161]: Could we throw an exception if there is a hyperparameter type that is not covered?

File: syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+    assert scenario.startswith("yahpo")
+    scenario = scenario[6:]
+
+    local_config.init_config()
+    local_config.set_data_path(str(repository_path / "yahpo"))
+
+    # Select a Benchmark, active_session False because the ONNX session can not be serialized.
+    bench = benchmark_set.BenchmarkSet(scenario, active_session=False)
+
+    # FIXME: I can now create one scenario per instance but this is probably
```
Comment (aaronkl) [inline syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py:190]: This seems like a left-over comment?

**Review by aaronkl — APPROVED** [review]
LGTM

**Comment by geoalgo** [comment]
@aaronkl I addressed your comments, can you approve again?

**Review by aaronkl — APPROVED** [review]

**Pull request merged**